src/redux/reducers/resources.js

```diff
@@ -125,8 +134,27 @@ export default handleActions(
         inSourceTarget: resourceHierarchy
       };
     },
-    // Open/Close Container Resources in UX
+    /**
+     * Untrack API call and track failure that occurred.
+     * Dispatched via Saga call on failed Resource Collection call.
+     **/
+    [actionCreators.resources.loadFromSourceTargetFailure]: (state, action) => ({
+      ...state,
+      pendingAPIResponse: false,
+      pendingAPIOperations: untrackAction(
+        actionCreators.resources.loadFromSourceTarget,
+        state.pendingAPIOperations,
+      ),
+      apiOperationErrors: [...state.apiOperationErrors, {
+        'action': 'source_resource_collection',
```

Comment:
Maybe change this and all references to this hardcoded string to
`actionCreators.resources.loadFromSourceTarget.toString()`